docs/upgrade/11_to_12/workflows_postgresql.py

```diff
@@ -0,0 +1,427 @@
+# Upgrade script for PostgreSQL databases
```

Comment:
I'm leaving this out since the comments on this would be very much the same as the one on the MariaDB script